photowall/app/elements/presentation-api/presentation-api.html

```diff
@@ -96,8 +96,8 @@
        * @function
        */
       join: function() {
-        var that = this;
-        if (this.isConnected) {
+        const that = this;
+        if (this.isConnected) {;
```

Comment:
I decided to change all that vars to either const (or let, where appropriate) following ES6 specs. Removed the erroneous semi-colon!